functionsSrc/addToNewsletterList/addToNewsletterList.js

```diff
@@ -0,0 +1,57 @@
+const request = require('request');
+
+module.exports.handler = (event, context, callback) => {
+  const { email } = event.queryStringParameters;
+  let errorMessage = null;
+
+  if (!email) {
+    errorMessage = 'No EMAIL supplied';
+    console.log(errorMessage);
+    callback(errorMessage);
+  }
+
+  const data = {
+    email_address: email,
+    status: 'subscribed',
+    merge_fields: {},
+  };
+
+  const subscriber = JSON.stringify(data);
+  console.log('Sending data to mailchimp', subscriber);
+
+  request({
+    method: 'POST',
+    url: 'https://us4.api.mailchimp.com/3.0/lists/cf9e550f84/members',
+    body: subscriber,
+    headers: {
+      Authorization: 'Basic cmFuZG9tc3R1ZGlvOmExOTA0MjJjZTdhMTdhM2UwOThhMTU1YzJlOTY2NjY0LXVzNA==',
```

Comment:
Hmm, this might be bad. Remember, this whole codebase is open source. Might have to stuff this into a Netlify secret.

Reply:
Good spot – I'll get Mark to rescind and generate a new one, and stick in an env variable